src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -261,6 +270,18 @@ class PeerActor(
 
       case GetStatus =>
         sender() ! StatusResponse(Handshaked)
+
+      case StartFastSync(targetHash) =>
+        //todo get storage form somewhere?
+        val storage = FastSyncActor.Storage(
+          new BlockHeadersStorage(EphemDataSource()),
```

Comment:
@adamsmo Actually I did a lot of _wiring_ work in order to have the `GetBlockByid` so it might be better not to change it. Will tell more during the VSU  :smile: